docs/config.rb

```diff
@@ -28,6 +28,7 @@
 configure :development do
   activate :gzip
   activate :livereload
+  activate :syntax
```

Comment:
You need to put this at the root level so that both build and dev have syntax highlighting